cfgov/unprocessed/apps/youth-employment-success/js/route-option-view.js

```diff
@@ -0,0 +1,106 @@
+import { checkDom, setInitFlag } from '../../../js/modules/util/atomic-helpers';
+import inputView from './input-view';
+
+const CLASSES = Object.freeze( {
+  FORM: 'o-yes-route-option',
+  TRANSPORTATION_CHECKBOX: 'a-yes-route-mode',
+  QUESTION_INPUT: 'a-yes-question'
+} );
+
+const defaultState = {
+  selectedTransportation: [],
+  daysPerWeek: '',
+  miles: '',
+  dailyCost: ''
+};
+
+/**
+ * RouteOptionFormView
+ * @class
+ *
+ * @classdesc Initializes the organism.
+ *
+ * @param {HTMLNode} element
+ *  The root DOM element for this view
+ * @returns {Object} The view's public methods
+ */
+function RouteOptionFormView( element ) {
+  let state = { ...defaultState };
+
+  const _dom = checkDom( element, CLASSES.FORM );
+  const _transportationTypes = Array.prototype.slice.call(
+    _dom.querySelectorAll( `.${ CLASSES.TRANSPORTATION_CHECKBOX }` )
+  );
+  const _textInputs = Array.prototype.slice.call(
+    _dom.querySelectorAll( `.${ CLASSES.QUESTION_INPUT }` )
+  );
+
+  /**
+   * Updates form state from child <input text> nodes
+   * @param {String} name Identifier of the piece of state to update
+   * @param {Object} event DOM event object
+   */
+  function _setQuestionResponse( name, event ) {
+    state = {
+      ...state,
+      [name]: event.target.value
+    };
+  }
+
+  /**
+   * Updates state from child <input checkbox> nodes
+   * @param {*} _ unused
```

Comment:
Why's this included if it's unused?

Reply:
The linter was complaining about this param not being noted in the jsdoc

Reply:
I mean why's the param there at all?